github/billing.go

```diff
@@ -44,6 +44,25 @@ type StorageBilling struct {
 	EstimatedStorageForMonth     int     `json:"estimated_storage_for_month"`
 }
 
+// ActiveCommitters represents the total active committers across all repositories in an Organization.
+type ActiveCommitters struct {
+	TotalAdvancedSecurityCommitters int            `json:"total_advanced_security_committers"`
+	Repositories                    []Repositories `json:"repositories"`
+}
+
+// Repositories represents active committers on each repository
```

Comment:
I think `Repositories` is too general and can be easily confused with `Repository`.

```suggestion
// RepositoryActiveCommitters represents active committers on each repository.
```